fundingmanager.go

```diff
@@ -1397,7 +1396,9 @@ func (f *fundingManager) waitForFundingWithTimeout(completeChan *channeldb.OpenC
 				return
 			}
 
-			if uint32(epoch.Height) >= maxHeight {
+			// If we are not the channel initiator it's safe
+			// to timeout the channel
+			if uint32(epoch.Height) >= maxHeight && !completeChan.IsInitiator {
```

Comment:
Method's `godoc` should be updated to reflect this.

Reply:
Why not just not launch this method at all, and only go with the `waitForFundingConfrimation`? Also if one is the initiator, and an amount was pushed to them, then seems that they should also wait as they have funds at stake.